docs/source/tensor_attributes.rst

```diff
@@ -15,23 +15,32 @@ torch.dtype
 .. class:: torch.dtype
 
 A :class:`torch.dtype` is an object that represents the data type of a
-:class:`torch.Tensor`. PyTorch has eleven different data types:
+:class:`torch.Tensor`. PyTorch has twelve different data types:
 
-========================   ===========================================   ===========================
+========================== ===========================================   ===========================
 Data type                  dtype                                         Legacy Constructors
-========================   ===========================================   ===========================
+========================== ===========================================   ===========================
 32-bit floating point      ``torch.float32`` or ``torch.float``          ``torch.*.FloatTensor``
 64-bit floating point      ``torch.float64`` or ``torch.double``         ``torch.*.DoubleTensor``
 64-bit complex             ``torch.complex64`` or ``torch.cfloat``
-128-bit floating point     ``torch.complex128`` or ``torch.cdouble``
-16-bit floating point      ``torch.float16`` or ``torch.half``           ``torch.*.HalfTensor``
+128-bit complex            ``torch.complex128`` or ``torch.cdouble``
+16-bit floating point [1]_ ``torch.float16`` or ``torch.half``           ``torch.*.HalfTensor``
+16-bit floating point [2]_ ``torch.bfloat16``                            ``torch.*.BFloat16Tensor``
 8-bit integer (unsigned)   ``torch.uint8``                               ``torch.*.ByteTensor``
 8-bit integer (signed)     ``torch.int8``                                ``torch.*.CharTensor``
 16-bit integer (signed)    ``torch.int16`` or ``torch.short``            ``torch.*.ShortTensor``
 32-bit integer (signed)    ``torch.int32`` or ``torch.int``              ``torch.*.IntTensor``
 64-bit integer (signed)    ``torch.int64`` or ``torch.long``             ``torch.*.LongTensor``
 Boolean                    ``torch.bool``                                ``torch.*.BoolTensor``
-========================   ===========================================   ===========================
+========================== ===========================================   ===========================
+
+.. [1] Sometimes referred to as binary16: uses 1 sign, 5 exponent, and 9 fraction
```

Comment:
10 fraction bits, denorm to achieve 11.

Reply:
Sorry, not denorm to achieve 10 fraction bits. All floating point formats (including bfloat16) are using implicit bit to "increase" the number of mantissa bits. Denorm numbers may or may not be supported (and possibly they are not supported in bfloat16).

Reply:
fixed, thanks